tools/azure-rest-api-specs-examples-automation/dotnet/main.py

```diff
@@ -314,20 +347,15 @@ def main():
 
     release = Release(config["release"]["tag"], config["release"]["package"], config["release"]["version"])
 
-    # samples/Generated/Samples
+    # find paths that contain the dotnet examples
     module_relative_path_local = get_module_relative_path(release.package, sdk_path)
-    dotnet_examples_relative_path = path.join(module_relative_path_local, "samples", "Generated", "Samples")
-    dotnet_examples_path = path.join(sdk_path, dotnet_examples_relative_path)
-    if not path.exists(dotnet_examples_path):
-        # fallback to tests/Generated/Samples
-        dotnet_examples_relative_path = path.join(module_relative_path_local, "tests", "Generated", "Samples")
-        dotnet_examples_path = path.join(sdk_path, dotnet_examples_relative_path)
+    dotnet_examples_paths = find_dotnet_examples_paths(sdk_path, module_relative_path_local)
```

Comment:
`find_dotnet_examples_paths` seems not check `samples` folder. So, any new .NET SDK release would have the new folder structure (i.e., under `tests` folder)?